setup/setup.py

```diff
@@ -37,7 +37,7 @@ def get_dirlist(_rootdir):
 
 setuptools.setup(
     name='nuscenes-devkit',
-    version='1.1.0',
+    version='1.1.1',
     author='Holger Caesar, Oscar Beijbom, Qiang Xu, Varun Bankiti, Alex H. Lang, Sourabh Vora, Venice Erin Liong, '
            'Sergi Widjaja, Kiwoo Shin, Caglayan Dicle, Freddy Boulton, Whye Kit Fong, Asha Asvathaman et al.',
     author_email='nuscenes@nutonomy.com',
```

Comment:
Does the email need to be changed to the Motional one?

Reply:
Can